src/App.tsx

```diff
@@ -1,24 +1,274 @@
-/* eslint-disable max-len */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-param-reassign */
+/* eslint-disable jsx-a11y/no-autofocus */
+/* eslint-disable no-useless-return */
```

Comment:
I don't you should have some many `eslint-disable` in your code - at some point it will stop to make sense to even use it